index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Figma to HTML & CSS</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;800&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    <div class="app"> 
+        <nav class="nav-bar">
+            <ul class="nav-bar-list">
+                <li class="nav-link">
+                    <a class="nav-link-text" href="#">
+                       <img src="images/app-logo.png" alt="logo" />
+                    </a>
+                </li>
+                
+                <li class="nav-link"><a class="nav-link-text" href="#">About</a></li>
+                <li class="nav-link"><a class="nav-link-text" href="#">Products</a></li>
+                <li class="nav-link"><a class="nav-link-text" href="#">Pricing</a></li>
+                <li class="nav-link"><a class="nav-link-text" href="#">Blog</a></li>
+                <li class="nav-link"><a class="nav-link-text" href="#">Jobs</a></li>
+                <li class="nav-link"><a class="nav-link-text" href="#">More</a></li>
+                </ul>
+                <ul class="nav-bar-list-right">
+                
+                <li class="nav-link">
+                    <a class="nav-link-text" href="#">
+                       <img src="images/google-play-badge.png" alt="Google Play Badge" />
+                    </a>
+                </li>
+                
+                <li class="nav-link">
+                    <a class="nav-link-text" href="#">
+                       <img src="images/app-store-badge.png" alt="App Store Badge" />
+                    </a>
+                </li>
+            </ul>
+        </nav>
+        <section class="hero-section">
+            <div class="hero-content-wrapper">
+                <div class="hero-content">
+                     <h1 class="hero-title">Organise Projects.<br>
+                         Get More Done.
+                    </h1>
+                    <button class="hero-button"> Get Started </button>
+                </div>
+                <div class="hero-image-container">
+                    <img src="images/phone-mockup.png" alt="hero image"/>
+                </div>
+                <div class="hero-company-logos">
```

Comment:
`.hero-company-logos` `.hero-image-container` ve `.hero-content` 'in hepsi hero-content-wrapper'ın içerisinde. Ve hero-content-wrapper'ı flex yaptığın için hepsi yan yana duruyor. Onun yerine ya content ve image container'ı aynı div in içerisine alıp aldığın div'i flex yapacaksın. Ya da .hero-company-logos 'u  hero-content-wrapper'ın dışına çıkartmalısın